optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/ProblemBasedSingleStatistic.java

```diff
@@ -34,4 +34,9 @@ public ProblemStatisticType getStatisticType() {
         return problemStatisticType;
     }
 
+    @Override
+    public String toString() {
+        return singleBenchmarkResult.toString() + "_" + problemStatisticType.toString();
```

Comment:
when concatening strings with +, it's not needed to do .toString().
This gives the same result: 

    return singleBenchmarkResult + "_" + problemStatisticType;

Furthermore, it even avoids a nullpointerexception if for example problemStatisticType is null (which it never is in this particular case)